frontend/src/apis/index.js

```diff
@@ -0,0 +1,39 @@
+// Is an abstraction to communicate with backend
```

Comment:
It would be good if the comments were formatted like `Abstraction for communication with backend` or `Abstracts away the communication with backend`. Also it seems quite clear what this class is doing so might not need anything if things are just named appropriately.